gems/manageiq_ansible_tower/lib/manageiq_ansible_tower.rb

```diff
@@ -0,0 +1,15 @@
+require "manageiq_ansible_tower/version"
+require "manageiq_ansible_tower/logging"
+
+require "manageiq_ansible_tower/connection"
+require "manageiq_ansible_tower/host"
+
+module ManageiqAnsibleTower
+  class << self
+    attr_accessor :logger
```

Comment:
If you're going to create a logger getter anyway, you may just want to make this an attr_writer.